packages/storybook/src/stories/1-MarkdownEditor.stories.js

```diff
@@ -93,28 +92,42 @@ export const Demo = () => {
    * Called when the markdown changes
    */
   const onMarkdownChange = useCallback((markdown) => {
-    localStorage.setItem('markdown-editor', markdown);
+    localStorage.setItem("markdown-editor", markdown);
   }, []);
 
   /**
    * Called when the Slate Value changes
    */
   const onSlateValueChange = useCallback((slateChildren) => {
-    action('onSlateValueChange')(slateChildren);
-    localStorage.setItem('slate-editor-value', JSON.stringify(slateChildren));
-    const slateValue = {
+    /*
+      LocalStorage sets the Changed value
+    */
+    localStorage.setItem("slate-editor-value", JSON.stringify(slateChildren));
+
+    /*
+      setMarkdown and setSlateValue are state-hooks which deal with handling data with Markdown
+    */
+    setMarkdown(markdown);
+    /* Serializing MarkDown Points */
+    const SlateValue = {
       document: {
-        children: slateChildren
-      }
+        children: slateChildren,
+      },
     };
-    const markdown = slateTransformer.toMarkdown(slateValue);
-    setSlateValue(slateValue.document.children);
-    action('markdown')(markdown);
-    setMarkdown(markdown);
-  }, []);
+    setSlateValue(slateChildren);
+    const edited_markdown = slateTransformer.toMarkdown(SlateValue);
+
+    /*
+    Actions are a feature of stories which allow live changes to be recorded in an action.
+    (https://storybook.js.org/docs/react/essentials/actions#gatsby-focus-wrapper)
+    */
+
+    action("onSlateValueChange")(slateChildren);
+    action("setMarkdown")(edited_markdown);
+  });
```

Comment:
@dselman If you see here, I changed how the way Markdown is rendered, it previously didn't render the ordered lists correctly. After these changes, It renders correctly.  I have also mentioned the docs related to storybook